package.json

```diff
@@ -152,6 +152,7 @@
     "istanbul-instrumenter-loader": "^3.0.1",
     "istanbul-lib-coverage": "^3.0.0",
     "jest": "^27.0.6",
+    "jest-fetch-mock": "^3.0.3",
```

Comment:
I could have used `nock` for this as well, but it's a lower-level mock that works for any request client while `jest-fetch-mock` is a fetch-specific mock that operates at the fetch `Response` level rather than the raw request level. I used `fetch` here rather than something like `superagent` (which is used elsewhere in CLUE) because now that `fetch` is widely available in browsers it seems silly to continue to use multiple different clients (`axios`, `superagent`, etc.) in different CC projects when we can simply standardize on what's available in the browser. Given that I'm using fetch, it seemed to make sense to use a fetch-specific mock.